wire/internal/wire/wire_test.go

```diff
@@ -81,15 +82,22 @@ func TestWire(t *testing.T) {
 				defer t.Logf("wire_gen.go:\n%s", gen)
 			}
 			if len(errs) > 0 {
-				for _, e := range errs {
-					t.Log(e)
+				gotErrStrings := make([]string, len(errs))
+				for i, e := range errs {
+					gotErrStrings[i] = scrubError(e.Error())
+					t.Log(gotErrStrings[i])
 				}
 				if !test.wantWireError {
-					t.Fatal("Did not expect errors.")
+					t.Fatal("Did not expect errors. To -record an error, create want/wire_errs.txt.")
 				}
-				for _, s := range test.wantWireErrorStrings {
-					if !errorListContains(errs, s) {
-						t.Errorf("Errors did not contain %q", s)
+				if *setup.Record {
+					wireErrsFile := filepath.Join(testRoot, test.name, "want", "wire_errs.txt")
+					if err := ioutil.WriteFile(wireErrsFile, []byte(strings.Join(gotErrStrings, "\n\n")), 0666); err != nil {
```

Comment:
This feels unfortunate, since unlike the expected output, the errors is an intentional minimum bound on what's expected instead of verbatim. This behavior would clobber existing files and the dev would have to go back and clean up diffs.

Reply:
Actually, with this change, `wire_errs.txt` is not a minimum bound anymore. It's exactly the set of errors that are output (well, scrubbed). So, I think `-record` makes perfect sense.

Do you disagree with the change from it being an intentional minimum bound?

Reply:
Oh I didn't realize the change. Yes, I suppose that if the line numbers are scrubbed and the paths are stable, having exact error output is fine.